### PR TITLE
added swipes data

### PR DESCRIPTION
**Pull request by AAAstorga**
I added swipe functionality so if you have a meal plan with swipe data, it will show up in eatery

### CODE REVIEW

File: src/schema.py [hunk 1]
```diff
@@ -100,16 +100,16 @@ def resolve_account_info(self, info, session_id=None):
     account_info['brbs'] = '0.00'
     account_info['city_bucks'] = '0.00'
     account_info['laundry'] = '0.00'
+    account_info['swipes'] = ''
```
Comment (young-k) [inline src/schema.py:105]: shouldn't this be initialized to `0`?

File: src/schema.py [hunk 2]
```diff
@@ -100,16 +100,16 @@ def resolve_account_info(self, info, session_id=None):
...
       if acct['accountDisplayName'] == ACCOUNT_NAMES['citybucks']:
         account_info['city_bucks'] = str("{0:.2f}".format(round(acct['balance'], 2)))
       elif acct['accountDisplayName'] == ACCOUNT_NAMES['laundry']:
         account_info['laundry'] = str("{0:.2f}".format(round(acct['balance'], 2)))
       elif ACCOUNT_NAMES['brbs'] in acct['accountDisplayName']:
         account_info['brbs'] = str("{0:.2f}".format(round(acct['balance'], 2)))
-      # Need more research to implement swipes:
-      # Each plan has a different accountDisplayName
-      account_info['swipes'] = ''
+      elif any(meal_swipe_name in acct['accountDisplayName'] for meal_swipe_name in ['Bear Basic', 'Bear Choice', 'Bear Traditional']):
```
Comment (young-k) [inline src/schema.py]: why not define `['Bear Basic', 'Bear Choice', 'Bear Traditional']` in a constant up top?

also, i am not a fan of this nested one-line for loop, but we can verify with @meganle

**Comment by young-k** [comment]
Your commit message should also be `Add swipes data`

File: src/schema.py (hunk 1 above)
Comment (AAAstorga) [reply src/schema.py:105]: No sir

File: src/schema.py (hunk 2 above)
Comment (AAAstorga) [reply src/schema.py]: Not sure. I don't really do python backend, was just trying to fix a feature

Comment (meganle) [reply src/schema.py]: yeah, i think it's best if the array is a constant.

also is there any reason that you use `any(...)`?

Comment (AAAstorga) [reply src/schema.py]: stackoverflow

**Comment by ilikesymmetry** [comment]
@young-k @meganle @keivanshahida @sahamanish99

**Review by meganle — APPROVED** [review]

**Comment by keivanshahida** [comment]
This looks good to me - pretty straightforward changes.

**Review by Omarrasheed — APPROVED** [review]
LGTM fam -def backend dev

**Pull request merged**